xclim/sdba/adjustment.py

```diff
@@ -221,7 +221,14 @@ def adjust(self, sim: DataArray, *args, **kwargs):
             if name in scen.coords:
                 scen[name].attrs.update(crd.attrs)
 
-        params = ", ".join([f"{k}={repr(v)}" for k, v in kwargs.items()])
+        params = ", ".join(
+            [
+                f"{k}={repr(v)}"
+                if not isinstance(v, xr.DataArray)
+                else f"{k}=xarray.DataArray '{v.name}'"
+                for k, v in kwargs.items()
+            ]
+        )
```

Comment:
What do you say of :
```suggestion
        params = gen_call_string('', **kwargs)[1:-1]  # indexing to remove added ( )
```
(imported from `xclim.core.formatting`)
This performs some other checks and prints shorter strings for lists and dicts. For named DataArrays you'll get `<name>` and <array> for nameless.